Header header
uint8[] left_hand_position
uint8[] right_hand_position